egs2/sdsv_2021/spk1/README.md

```diff
@@ -0,0 +1,14 @@
+# RESULTS
+Overall results
+
+## Test set: Vox1-O
```

Comment:
Please add config and model link.
Please refer to https://github.com/espnet/espnet/blob/master/egs2/voxceleb/spk1/README.md and other recipes.